test/RoslynAnalyzer.Tests/RequiresUnreferencedCodeTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Analyzer1.Test.CSharpAnalyzerVerifier<
+    ILTrimmingAnalyzer.RequiresUnreferencedCodeAnalyzer>;
+
+namespace ILTrimmingAnalyzer.Test
+{
+    public class RequiresUnreferencedCodeTest
+    {
+        [Fact]
+        public Task SimpleDiagnostic()
+        {
+            var src = @"
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Looking forward - is there some typical way how these are written as .cs files so that writing them can use intellisense and so on?
Not sure how important that is for Roslyn, but in our case all of these snippets will be perfectly valid C#.

Reply:
Nope, no way.